pyV2DL3/eventdisplay/fillEVENTS.py

```diff
@@ -168,6 +168,15 @@ def __fillEVENTS__(edFileIO, select=None):
 
         avPedvar = runSummary["pedvarsOn"][0]
 
+        wobble = np.array([
```

Comment:
Why do you read about the wobble directions for tree entry 0 and 1? runSummary contains run parameters per run (entry  0 to N-1) and the combined values (last entry; always 0. for the wobble offset, as the combination of runs don't have a wobble offset). 

We are probably looking in most cases with V2DL3 at anasum files with one run only - so you probably need only Entry 0.